### ISSUE TITLE
Can't join the arena with a username containing non US-ASCII charackter

### ISSUE THREAD

**Issue by tarczynskitomek** [GoogleCloudPlatform/cloudbowl-microservice-game#24]
Tried joining with a user name "Tomek Tarczy**ń**ski" - note the ń char, and the validation didn't allow me.

Steps to reproduce:
1. Try joining the arena
2. Use username with with ń or ł char
3. Get a validation error
4. Change ń to n
5. Validation passes

**Comment by jamesward** [comment]
Yeah, that is just my terrible mistake.  The code is here: https://github.com/GoogleCloudPlatform/cloudbowl-microservice-game/blob/master/app/models/Data.scala#L470

I'm not quite sure why I restricted that field.  But this is definitely a bug I'll fix.  Thanks for reporting.

**Comment by tarczynskitomek** [comment]
Hi @jamesward, thanks for looking into this! I think the check is a reasonable thing it's just too restrictive. What I'd suggest is e.g. `[^\p{L}0-9\s]` regex - it'll match any characters other that space, numbers and any kind of letter from any language. I can create a PR if you think it's fine ;-)

**Comment by jamesward** [comment]
I think that sounds great and would love a PR :)

**Comment by tarczynskitomek** [comment]
[Done](https://github.com/GoogleCloudPlatform/cloudbowl-microservice-game/pull/28), also signed the CLA thingy :-)
